common/merkle_light/src/lib.rs

```diff
@@ -154,7 +154,6 @@
     unused_import_braces
 )]
 #![cfg_attr(not(feature = "std"), no_std)]
-#![cfg_attr(feature = "nightly", allow(unstable_features))]
```

Comment:
The feature `nightly` does not exist and no nightly feature is used by the crate